TESTING.md

```diff
@@ -0,0 +1,7 @@
+# Testing
+
+Run:
+
+```cmd
+python -m unittest discover -s tests
```

Comment:
this makes the template specific to Python projects? What about React and database?

Reply:
I replaced the specific files with the generic template and kept the workflows to enable checks. 

@SonOfLope, it would be very interesting to add a check that verifies all types of files present and ensures that the corresponding workflow is also present.